hooks/post/30-upgrade.rb

```diff
@@ -1,3 +1,3 @@
-if foreman_server?
+if !app_value(:noop) && [0, 2].include?(@kafo.exit_code) && foreman_server?
```

Comment:
`[0, 2].include?(@kafo.exit_code)` is such a crude way to say "did kafo finish successfully?", but this is not a new patern, so you get a pass ;)

Reply:
We do use that `[0, 2]` check in a few places, could we move it to the hook extensions as a function?

Reply:
How about we merge this PR now and I'll update kafo to provide that + update develop to use it.

Reply:
Deal, here's the Redmine -- https://projects.theforeman.org/issues/29112